README.md

```diff
@@ -45,6 +74,21 @@ use MarjovanLier\SouthAfricanIDValidator\SouthAfricanIDValidator;
 
 $idNumber = 'Your South African ID number here';
 
+## Troubleshooting
```

Comment:
I think there is an issue with the Troubleshooting section. Shouldn't the bash script be closed before the Troubleshooting section?